gdpr_assist/admin/base.py

```diff
@@ -10,6 +10,7 @@
 
 from django.conf.urls import url
 from django.contrib import admin
+from django.contrib.admin.helpers import ACTION_CHECKBOX_NAME
```

Comment:
Compatibility import of was `ACTION_CHECKBOX_NAME` removed: https://docs.djangoproject.com/en/3.2/_modules/django/db/models/query_utils/